src/Views/Appearance.vala

```diff
@@ -227,9 +258,79 @@ public class PantheonShell.Appearance : Gtk.Grid {
                 pantheon_act.prefers_color_scheme = Granite.Settings.ColorScheme.NO_PREFERENCE;
             });
 
+            prefer_default_radio.pressed.connect (() => {
+                schedule_mode_button.selected = 0;
+            });
+
             prefer_dark_radio.toggled.connect (() => {
                 pantheon_act.prefers_color_scheme = Granite.Settings.ColorScheme.DARK;
             });
```

Comment:
Button pressed is deprecated and we should make sure to connect to both of these and add a comment so it's clear why we're not connecting to the toggle event:

```suggestion
            prefer_dark_radio.toggled.connect (() => {
                pantheon_act.prefers_color_scheme = Granite.Settings.ColorScheme.DARK;
            });
            
            /* Connect to button_release_event so that this is only triggered
             * through user interaction, not if scheduling changes the selection
             */
            prefer_default_radio.button_release_event.connect (() => {
                schedule_mode_button.selected = 0;
                return Gdk.EVENT_PROPAGATE;
            });

            prefer_dark_radio.button_release_event.connect (() => {
                schedule_mode_button.selected = 0;
                return Gdk.EVENT_PROPAGATE;
            });
```